record_adb.py

```diff
@@ -0,0 +1,83 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# Common functions used throughout FNPRMS
+
+
+
+
+import sys
+import argparse
+import subprocess
+import time
+import os
+import signal
+
+
+#This script records the phone either through intent or simulating user touch for the purpose of generating videos that are consistent.
+#The consistency allows us to compare said video.
+def main(args):
+    method = args.input
+    recording_name = './sdcard/output.mp4'
+
+    if args.output is not None:
+            recording_name = './sdcard/' + args.output
+    if method in 'touch' and (args.coordinate_x is None or args.coordinate_y is None):
+        print('--touch requires --coordinate-x <coordinate> --coordinate-y <coordinate> to use the touch input')
+        sys.exit()
+    if method in 'intent' and args.package is None:
+        print('--intent requires --package <your.package.name>')
+        sys.exit()
+
+    check_for_existing_process()
+    record_process = subprocess.Popen(['adb', 'shell','screenrecord'] + [recording_name])
+    time.sleep(3)
+
+    if method in 'intent':
+        record_with_intent(args.package)
+    else:
+        simulate_input(args.coordinate_x, args.coordinate_y)
+    pull_recording(record_process, recording_name)
+
+def record_with_intent(package):
+    activity_start = subprocess.Popen(['adb', 'shell', 'am', 'start-activity', package +'/.App', '--ez finishonboarding true'])
```

Comment:
We can't use `--ez finishonboarding` on an actual test run: it does additional processing and doesn't closely follow what users experience.

Again, I'd prefer if you'd just delete this code.

(that being said I'm not sure `.App` works in January but maybe it does)

Reply:
delete the whole `record_with_intent`? or just the `--ez onboarding` and .App works with Jan version :)

Reply:
I'd prefer to delete the whole thing so I don't need to think harder about whether not it works correctly. Alternatively, you can comment it out and add a "TODO: REVIEW THIS" or something so that we know to look at it.

Reply:
sounds good 👍